daemon/config.go

```diff
@@ -59,6 +59,7 @@ type CommonConfig struct {
 	LogConfig            LogConfig           `json:"log-config,omitempty"`
 	Mtu                  int                 `json:"mtu,omitempty"`
 	Pidfile              string              `json:"pidfile,omitempty"`
+	RawLogs              bool                `json:"raw-logs"`
```

Comment:
please, add `omitempty` so the config loader doesn't think it was set when the value is `false`.